models/reviews-models.js

```diff
@@ -55,20 +44,32 @@ exports.fetchReviews = (
     "category",
     "created_at",
     "votes",
-    "DESC",
-    "ASC",
   ];
 
-  const validCategories = ["euro game", "dexterity", "social deduction", ""];
+  const validOrderByQueries = ["DESC", "ASC"];
+
+  const validCategories = [
+    "euro game",
+    "dexterity",
+    "social deduction",
+    "children's games",
+    "",
+  ];
+
+  if (!validSortByQueries.includes(sort_by)) {
+    return Promise.reject({ status: 400, msg: "Invalid Query" });
+  }
 
-  if (!validQueries.includes(sort_by)) {
+  if (!validOrderByQueries.includes(order.toUpperCase())) {
     return Promise.reject({ status: 400, msg: "Invalid Query" });
   }
 
   if (!validCategories.includes(category)) {
     return Promise.reject({ status: 400, msg: "Invalid Query" });
   }
 
+  category = category.replace(`'`, `''`);
+
```

Comment:
This comment is actually for a bit of code below - rather than checking category.length - we could just pass in 'category'. If we have not passed on in, this will default to undefined and be a falsy value